README.md

```diff
@@ -2,3 +2,21 @@
 
 [Onboarding exercise](https://github.com/wexinc/ps-tag-onboarding) for Tag team.
 
+## Installation
+Make sure you have the following mandatory software installed:
+- Docker && Docker-compose
+
+To run the application, you only need to run the docker-compose build and run.   
+```docker-compose up -d```   
+
+Server is running default at localhost:8080, you can change in docker-compose.yml
+
+## API Documentation
+You can acess the Swagger API docs with the application running.   
+Click here -> [Documentation](http://localhost:8080/swagger/index.html)
+
+
+## Technologies Used
+- Go 1.21
```

Comment:
Add here the swagger as technology used.
If you needed to install something, add the link to the installation manual too